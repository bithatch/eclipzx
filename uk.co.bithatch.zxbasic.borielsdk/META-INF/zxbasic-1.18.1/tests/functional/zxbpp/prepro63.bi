

#define q(a)XX##a

q(5)
