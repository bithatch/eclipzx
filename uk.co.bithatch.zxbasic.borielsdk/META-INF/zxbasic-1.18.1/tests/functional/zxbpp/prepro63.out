#line 1 "prepro63.bi"


#line 4 "prepro63.bi"

XX5
